### DISCUSSION TITLE
3D Keypoints - Units and Scale Factor

### DISCUSSION THREAD

**Discussion by mjcpiopio** [open-mmlab/mmpose#2904] [Q&A]
Hello! I am using the 3D algorithms available, MotionBert, VideoPose3D and Simplebaseline. I wanted to compare the outputs of this algorithms by transforming the keypoints coordinates output in meters. How can I do that? What is the scale used in the output referential of these algorithms? thank you!

**Comment by HabibaAbderrahim** [comment]
I've posed similar question before but  I didn't receive an answer. Probably the unit is mm , I am making suggestions because the used dataset is  Human3.6 , if you have an updates pls let me know .
